### PR TITLE
Kata deploy automation

### PR DESCRIPTION
**Pull request by egernst**
simple script to update the Dockerfile for kata-deploy to a new version for a given branch.  based off of the update-repository-version script.

TODO:  add testing, similar to update-repository-version_test.sh

### CODE REVIEW

File: kata-deploy/Dockerfile [hunk 1]
```diff
@@ -1,5 +1,5 @@
 FROM centos/systemd
-ARG KATA_VER=1.2.0
+ARG KATA_VER=1.4.0
```
Comment (jodh-intel) [inline kata-deploy/Dockerfile]: Didn't you just fix this on #252?

File: release/bump-kata-deploy.sh [hunk 2]
```diff
@@ -0,0 +1,111 @@
...
+	h) usage 0 ;;
+	p) PUSH="true" ;;
+	esac
+done
+
+shift $((OPTIND - 1))
+
+new_version=${1:-}
+target_branch=${2:-}
+[ -n "${new_version}" ] || (echo "ERROR: no new version" && usage 1)
```
Comment (jodh-intel) [inline release/bump-kata-deploy.sh]: Why not use `die()` here?

```
[ -n "${new_version}" ] || die "no new version"
```

File: kata-deploy/Dockerfile [hunk 3]
```diff
@@ -1,5 +1,5 @@
...
 ARG ARCH=x86_64
```
Comment (jodh-intel) [inline kata-deploy/Dockerfile]: Not part of your PR, but why are we hard-coding the architecture?

**Comment by jodh-intel** [comment]
Your PR description contains a TODO so should this have a `dnm` tag? If not, please convert those TODO items into new issues.

**Comment by jodh-intel** [comment]
Ping @egernst.

**Comment by caoruidong** [comment]
@egernst ping

**Comment by egernst** [comment]
thanks for the review @jodh-intel and the follow up @caoruidong .

I'm going to start working on this again now.

**Comment by jodh-intel** [comment]
Ping @egernst.

**Comment by jodh-intel** [comment]
Re-ping @egernst.

**Pull request closed without merging**